src/lora/lorawan_mac_region.erl

```diff
@@ -211,7 +211,7 @@ max_payload_size(Region, DR) ->
         'US915' -> maps:get(DR, ?US915_PAYLOAD_SIZE_MAP, ?US915_MAX_DOWNLINK_SIZE);
         'AU915' -> maps:get(DR, ?AU915_PAYLOAD_SIZE_MAP, ?AU915_MAX_DOWNLINK_SIZE);
         'EU863' -> maps:get(DR, ?EU863_PAYLOAD_SIZE_MAP, ?EU863_MAX_DOWNLINK_SIZE);
-        _ -> maps:get(DR, ?US915_PAYLOAD_SIZE_MAP, ?US915_MAX_DOWNLINK_SIZE)
+        _ -> maps:get(DR, ?EU868_PAYLOAD_SIZE_MAP, ?EU868_MAX_DOWNLINK_SIZE)
```

Comment:
just curious, why EU868 as default?

Reply:
The EU868 values are the most "generic".  Most other regions are just slight varations of EU868.  If we are missing the Region value, then this default should be the safest choice.